openhcl/underhill_core/src/nvme_manager.rs

```diff
@@ -89,18 +91,20 @@ impl NvmeManager {
     pub fn new(
         driver_source: &VmTaskDriverSource,
         vp_count: u32,
-        dma_buffer_spawner: Box<dyn Fn(String) -> anyhow::Result<Arc<dyn VfioDmaBuffer>> + Send>,
+        _dma_buffer_spawner: Box<dyn Fn(String) -> anyhow::Result<Arc<dyn VfioDmaBuffer>> + Send>,
         save_restore_supported: bool,
         saved_state: Option<NvmeSavedState>,
+        dma_manager: GlobalDmaManager,
```

Comment:
Passing this by value raises questions as noted in the `Clone` derive for GlobalDmaManager.